game/g_session.cpp

```diff
@@ -121,6 +121,7 @@ result session::init (string::view cmdline)
 
     // sound indices are shared over the network so sounds
     // need to be registed in the same order on all clients
+
```

Comment:
nit: whitespace

Reply:
I saw this as I double-checked my revised PR and I _knew_ you would say something!

fixed :^))) and because you didn't ask for it, I fixed your typo too! 😘